docs/source/concept_models/vivarium_census_synthdata/reference_person_update_relationship_mapping.csv

```diff
@@ -0,0 +1,241 @@
+Sibling,Opp-sex spouse,Other relative,
+Parent,Opp-sex spouse,Parent-in-law,
+Grandchild,Opp-sex spouse,Grandchild,
+Parent-in-law,Opp-sex spouse,Parent,
+Child-in-law,Opp-sex spouse,Child-in-law,
+Other relative,Opp-sex spouse,Other relative,
+Roommate,Opp-sex spouse,Roommate,Questionable how people code roommate vs other nonrelative
+Foster child,Opp-sex spouse,Foster child,
+Other nonrelative,Opp-sex spouse,Other nonrelative,
+Biological child,Opp-sex partner,Biological child,
+Adopted child,Opp-sex partner,Adopted child,
+Stepchild,Opp-sex partner,Biological child,Could be adopted child
+Sibling,Opp-sex partner,Other relative,
+Parent,Opp-sex partner,Other nonrelative,
+Grandchild,Opp-sex partner,Grandchild,
+Parent-in-law,Opp-sex partner,Parent,"This likely shouldn't happen; we assume it is due to a liberal use of ""in-law"""
+Child-in-law,Opp-sex partner,Child-in-law,
+Other relative,Opp-sex partner,Other nonrelative,
+Roommate,Opp-sex partner,Roommate,Questionable how people code roommate vs other nonrelative
+Foster child,Opp-sex partner,Foster child,
+Other nonrelative,Opp-sex partner,Other nonrelative,
+Biological child,Same-sex spouse,Biological child,
+Adopted child,Same-sex spouse,Adopted child,
+Stepchild,Same-sex spouse,Biological child,Could be adopted child
+Sibling,Same-sex spouse,Other relative,
+Parent,Same-sex spouse,Parent-in-law,
+Grandchild,Same-sex spouse,Grandchild,
+Parent-in-law,Same-sex spouse,Parent,
+Child-in-law,Same-sex spouse,Child-in-law,
+Other relative,Same-sex spouse,Other relative,
+Roommate,Same-sex spouse,Roommate,Questionable how people code roommate vs other nonrelative
+Foster child,Same-sex spouse,Foster child,
+Other nonrelative,Same-sex spouse,Other nonrelative,
+Biological child,Same-sex partner,Biological child,
+Adopted child,Same-sex partner,Adopted child,
+Stepchild,Same-sex partner,Biological child,Could be adopted child
+Sibling,Same-sex partner,Other relative,
+Parent,Same-sex partner,Other nonrelative,
+Grandchild,Same-sex partner,Grandchild,
+Parent-in-law,Same-sex partner,Parent,"This likely shouldn't happen; we assume it is due to a liberal use of ""in-law"""
+Child-in-law,Same-sex partner,Child-in-law,
+Other relative,Same-sex partner,Other nonrelative,
+Roommate,Same-sex partner,Roommate,Questionable how people code roommate vs other nonrelative
+Foster child,Same-sex partner,Foster child,
+Other nonrelative,Same-sex partner,Other nonrelative,
+Opp-sex spouse,Biological child,Parent,
+Opp-sex partner,Biological child,Parent,
+Same-sex spouse,Biological child,Parent,
+Same-sex partner,Biological child,Parent,
+Biological child,Biological child,Sibling,"Could be half-sibling, which might not be reported as sibling"
+Adopted child,Biological child,Sibling,Might not be reported as sibling
+Stepchild,Biological child,Other relative,"Assuming step-siblings are not reported as ""brother or sister"""
+Sibling,Biological child,Other relative,
+Parent,Biological child,Other relative,

```

Comment:
Why is this other relative but row 60 is grandchild? I think this should also be grandchild. Same comment for adopted child below. Granted if we assign to the oldest this should never happen anyways

Reply:
Excellent catch -- row 60 is definitely wrong because it turns the parent-in-law _into_ the grandchild. I'm going to go back through these one more time and make sure I didn't do any more of them backward 😬

Reply:
Re: "assign to the oldest," yes good point this file is a little overkill. But, now we have it if we ever want to change that rule ¯\_(ツ)_/¯